src/entity/AdvancedSettings.ts

```diff
@@ -11,6 +11,9 @@ export class AdvancedSettings {
   @Column({ type: 'text' })
   description: string;
 
+  @Column({ type: 'text' })
```

Comment:
I think we should make the default value as part of the entity, so that when generating the migration (even though it is broken 😅) the relevant line should show the accurate result.

```suggestion
  @Column({ type: 'text', default: 'advanced' }) // ideally, if we can take the string to an enum, it would be better
```